package.json

```diff
@@ -27,6 +27,7 @@
     "docs": "npm run docs:lint && npm run docs:api",
     "jsdoc": "jsdoc",
     "docs:api": "jsdoc -c .jsdoc.json",
+    "postdocs:api": "node ./build/fix-api-docs.js",
```

Comment:
No, I think it's fine. Just wanted to verify that `prepublish` should work as we expect so the urls will get changed as part of that.